### DISCUSSION TITLE
how to use T-MAC in model

### DISCUSSION THREAD

**Discussion by eightfeetfish** [microsoft/T-MAC#11] [Q&A]
i want to use T-MAC  to accerate openAI Whisper model inferrence，anyone have a good idea?
Thanks

**Comment by kaleid-liner** [comment]
llama.cpp and [whisper.cpp](https://github.com/ggerganov/whisper.cpp) are both built upon [ggml](https://github.com/ggerganov/ggml). However, their code is evolving too fast, with a lot of refactoring. It seems the best option is to first apply our integration onto the latest ggml.

**Comment by zhouwg** [comment]
I don't understand why [your PR](https://github.com/ggml-org/llama.cpp/pull/13206) in the upstream llama.cpp is still suspending? Is it because that branch has conflicts with upstream?
